satviz-consumer/src/main/java/edu/kit/satviz/consumer/config/HeatmapColors.java

```diff
@@ -74,11 +67,11 @@ public boolean equals(Object o) {
       return false;
     }
     HeatmapColors colors = (HeatmapColors) o;
-    return fromColor == colors.fromColor && toColor == colors.toColor;
+    return hotColor.equals(colors.hotColor) && coldColor.equals(colors.coldColor);
```

Comment:
Objects.equals(x,y) > x.equals(y)

```suggestion
    return Objects.equals(hotColor, colors.hotColor) && Objects.equals(coldColor, colors.coldColor);
```

Reply:
Wurde angepasst.